### DISCUSSION TITLE
Docs: 在next14.2.5版本中，怎么监听路由的的变化

### DISCUSSION THREAD

**Discussion by Sunrisies** [vercel/next.js#67980] [Help]
### What is the update you wish to see?

在next14.2.5版本中，怎么监听路由的的变化

### Is there any context that might help us understand?

在next14.2.5版本中，怎么监听路由的的变化

**Comment by AnYiEE** [comment]
[文档](https://nextjs.org/docs/app/api-reference/functions/use-pathname)，怎么把issue当知乎呢😅。

This is not an issue, just a question, and it has been answered.
